_posts/fr/newsletters/2024-04-17-newsletter.md

```diff
@@ -0,0 +1,185 @@
+---
+title : 'Bulletin Hebdomadaire Bitcoin Optech #298'
+permalink : /fr/newsletters/2024/04/17/
+name: 2024-04-17-newsletter-fr
+slug: 2024-04-17-newsletter-fr
+type: newsletter
+layout: newsletter
+lang: fr
+---
+Le bulletin de cette semaine résume une analyse de la manière dont un nœud avec un mempool en
+cluster s'est comporté lorsqu'il a été testé avec toutes les transactions observées sur le réseau en
+2023. Nous incluons également nos sections régulières décrivant les mises à jour des clients et des services,
```

Comment:
This line starting with a number and . ("2023.") causes the formatting to be numbered bullet style.